Updates to tests including quarantining and tagging to fit the Docker requirements for automated testing. (#4782)
awsdocs/aws-doc-sdk-examples: php/example_code/README.md

Reviewer: Recommend updating this README according to the [Language level README template](https://github.com/awsdocs/aws-doc-sdk-examples/wiki/Language-level-README-template).

Author: Updated, moved, and re-reviewed. Thanks for this suggestion!

## Running the code 

By default, these code examples run using the default AWS credential provider chain, which includes using an AWS shared credentials file and profiles.
For more information, see [Using the AWS Credentials File and Credential Profiles](https://docs.aws.amazon.com/sdk-for-php/v3/developer-guide/guide_credentials_profiles.html) in the *AWS SDK for PHP Developer Guide*.

Running these code examples might result in charges to the AWS account that is associated with the AWS credentials being used.

To install dependencies, run `composer install` (https://getcomposer.org) before attempting to run example code.

Many examples come with a `Runner.php` file to abstract the logic of the example from running the code.
From any example directory with a `Runner.php` file, run the example with `php Runner.php`.

## Running the tests

Some of these code examples have accompanying unit and integration tests designed to run with PHPUnit.
These test files end with `Test.php` and are usually in a folder named `tests` next to the code examples themselves.

> They don't run on their own, but must be run via the `phpunit` command or your IDE.

Reviewer: ```suggestion
They don't run on their own, but must be run by using the `phpunit` command or your IDE.
```

To install dependencies, run `composer install` (https://getcomposer.org) before attempting to run test code.

To run the entire set of tests, run `phpunit` from the `php/example_code` directory.
To filter by a specific service, run `phpunit --testsuite <testsuite-name>` where `<testsuite-name>` is one of the options given with `phpunit --list-suites`.
To filter by either integration tests or unit tests, add `--group integ` or `--group unit` to the command.

> The suite and group filters may be used together or separate.

Reviewer: ```suggestion
The suite and group filters can be used together or separately.
```

For specific instructions on how to run individual tests, see the README files for each example.

## Docker image (Beta)
This example code will soon be available in a container image

> hosted on [Amazon Elastic Container Registry (ECR)](https://docs.aws.amazon.com/AmazonECR/latest/userguide/what-is-ecr.html).

Reviewer: ```suggestion
hosted on [Amazon Elastic Container Registry (Amazon ECR)](https://docs.aws.amazon.com/AmazonECR/latest/userguide/what-is-ecr.html).
```

> This image will be preloaded with all PHP examples with dependencies pre-resolved, allowing you to explore

Reviewer: ```suggestion
This image will be preloaded with all PHP examples with dependencies pre-resolved. This way, you can explore
```

these examples in an isolated environment.

The [SDK for PHP image](https://gallery.ecr.aws/b4v4v1s0/php) is available on ECR Public.

## Additional information

> - As an AWS best practice, grant these code examples "least privilege", or only the permissions required to perform a task. For more information, see 

Reviewer: ```suggestion
- As an AWS best practice, grant these code examples "least privilege," or only the permissions required to perform a task. For more information, see 
```
